protos/Robot_Echo/textures/License.md

```diff
@@ -0,0 +1,27 @@
+
+# Textures
+These textures are the property of Cyberbotics: 
```

Comment:
Would it be possible to either enumerate exactly which files this refers to and/or use a specific directory name for these?
Just in case we add any of our own textures here I think it's worth keeping it clear which are Cyberbotics' and which are not.

Reply:
I've moved the cyberbotics textures into a new sub-directory, within textures, and referenced the texture files within the license file.  Should be super clear if we add our own now